### PR TITLE
nym: Expand usability

### PR DESCRIPTION
**Pull request by lhh**
This expands the number of translations and enables the caller to (optionally) override the default translations.

### CODE REVIEW

File: toolchest/strutil.py [hunk 1]
```diff
@@ -46,8 +46,16 @@ def nym(arg):
         arg (string): A string to create the nym for
```
Comment (jguiditta) [inline toolchest/strutil.py:46]: issue: missing docstrings for the two new parameters.

**Review by jguiditta — COMMENTED** [review]
Overall, looks good, couple minor suggestions.

**Comment by jguiditta** [comment]
It looks to me like python 3.7 did not support the walrus operator, which is why we have a fail. However, 3.7 is EOL according to https://devguide.python.org/versions/. If nobody is opposed, I suggest we drop that from our support matrix, which I could do in a separate patch so this one is not affected.

**Comment by jguiditta** [comment]
Just realized this is still open. I did the fix I mentioned, this just needs a rebase and then should be good to go.

**Comment by lhh** [comment]
haha, I thought it was closed. Whoops.

**Review by jguiditta — APPROVED** [review]

**Review by jpichon — APPROVED** [review]

**Pull request merged**